skyportal/handlers/api/annotation.py

```diff
@@ -8,13 +8,32 @@
 
 class AnnotationHandler(BaseHandler):
     @auth_or_token
-    def get(self, annotation_id):
+    def get(self, associated_resource_type, resource_id, annotation_id):
         """
         ---
         description: Retrieve an annotation
         tags:
           - annotations
         parameters:
+          - in: path
+            name: associated_resource_type
+            required: true
+            schema:
+              type: string
+            description: |
+               What underlying data the annotation is on:
+               an "object" (default), or a "spectrum".
```

Comment:
I don't think the user has the option of using a "default" option?  It has to be `source` or `spectrum`, no?